Bottom: 140a99b7939b188805d00d27ad32b2e1b4a66f0d
Top:    1277600ae85f54de91680738ad22b8c0f223fbed
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-07 11:57:48 +0000

Debugging on again...


---

diff --git a/src/polygon1.c b/src/polygon1.c
index f2a2cce..9001b91 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -39,6 +39,7 @@
       are marked
 */
 
+#undef NDEBUG
 #include	<assert.h>
 #include	<stdlib.h>
 #include	<stdio.h>
